src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
If this is for VS only then we should only do this check when running VS tests. Otherwise our unit tests are unnecessarily suffering. 

To be clear: when we hit this limit it's a real failure that we will have 0 way of investigating. Hence this limit is equivalent to "this bug doesn't matter".